ios/Podfile

```diff
@@ -47,8 +53,7 @@ target "celo" do
     # Hermes is now enabled by default. Disable by setting this flag to false.
     # Upcoming versions of React Native may rely on get_default_flags(), but
     # we make it explicit here to aid in the React Native upgrade process.
-    # Disabled for now due to issues with Detox tests
-    :hermes_enabled => false,
+    :hermes_enabled => flags[:hermes_enabled],
```

Comment:
I'd argue we shouldn't enable hermes in this upgrade PR, but I also don't think it should block this from being merged. If we end up having an issue in CI we can revert similar to #3859.

Reply:
i'd like to enable hermes because it is a better user experience. i haven't noticed any issues with the CI so i'd like to leave it until we see issues